content/09.overfitting.md

```diff
@@ -28,7 +28,7 @@ Therefore, model performance should be evaluated with a carefully picked panel o
 When working with biological and medical data, one must also carefully consider potential sources of bias and/or non-independence when defining training and test sets.
 For example, a deep learning model for pneumonia detection in chest X-rays appeared to performed well within the hospitals providing the training data, but then failed to generalize to other hospitals [@doi:10.1371/journal.pmed.1002683].
 This resulted from the deep learning model picking up on signal related to which hospital the images were from, and represents a type of artifact or "batch effect" that practitioners must be vigilant towards.
-When dealing with sequence data, holding out test data that are evolutionarily related or that share structural homology to the training data can result in overfitting that is hard to detect due to the inherent relatedness of the partitioned data (cite?).
+When dealing with sequence data, holding out test data that are evolutionarily related or that share structural homology to the training data can result in overfitting that is hard to detect due to the inherent relatedness of the partitioned data [@10.1093/bib/bbv082].
```

Comment:
```suggestion
When dealing with sequence data, holding out test data that are evolutionarily related or that share structural homology to the training data can result in overfitting that is hard to detect due to the inherent relatedness of the partitioned data [@doi:10.1093/bib/bbv082].
```

The latest Manubot release supports citations like this (without the `doi:` prefix) but not the version we're using.